src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -389,6 +394,10 @@
         text: az aks update -g MyResourceGroup -n MyManagedCluster --api-server-authorized-ip-ranges ""
       - name: Restrict apiserver traffic in a kubernetes cluster to agentpool nodes.
         text: az aks update -g MyResourceGroup -n MyManagedCluster --api-server-authorized-ip-ranges 0.0.0.0/32
+      - name: Update a AKS-managed AAD cluster with tenant ID or admin group object IDs.
+        text: az aks update -g MyResourceGroup -n MyManagedCluster --aad-admin-group-object-ids <id-1,id-2> --aad-tenant-id <id>
```

Comment:
shouldn't it have --enable-aad?

Reply:
This example is to update a cluster whose managed AAD is already enabled. So it should not have --enable-aad.

There's another example below to enable AAD for the AADv1 cluster. This one should have --enable-aad.